apps/datafeeder/src/app/presentation/components/data-import-validation-map-panel/data-import-validation-map-panel.component.ts

```diff
@@ -22,11 +23,11 @@ export class DataImportValidationMapPanelComponent
   implements OnInit, AfterViewInit {
   @ViewChild('map') mapElt: ElementRef
 
-  @Input() mapViewId = 'map'
+  @Input() showProperties = false
   @Input() headerLabel = ''
   @Input() footerLabel = ''
   @Input() footerList = []
-  @Input() geoJson = ''
+  @Input() geoJson: any = {}
```

Comment:
type `Feature` from `geojson` types
I'm not sure it's necessary to give it a default value
`geoJson?: Feature`